dotcom-rendering/src/web/components/Accordion.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+import { body, breakpoints, from, space } from '@guardian/source-foundations';
+import type { FC } from 'react';
```

Comment:
[We should remove this and associated references](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/.eslintrc.js#L186-L196).